scanner/updater/manual/data.go

```diff
@@ -0,0 +1,24 @@
+package manual
+
+import (
+	"github.com/quay/claircore"
+)
+
+var manuallyEnrichedVulns = []*claircore.Vulnerability{
```

Comment:
Without much knowledge about Scanner or this process of updating, would it make sense to parse these vulnerabilities from a file, instead of defining them in code?